app/models/conversation.rb

```diff
@@ -25,57 +25,74 @@ class Conversation < ApplicationRecord
 
   after_create :send_events, :run_round_robin
 
-
   acts_as_taggable_on :labels
 
-  def update_assignee(agent=nil)
+  def update_assignee(agent = nil)
     self.assignee = agent
-    self.save!
+    save!
   end
 
-  def update_labels(labels=nil)
+  def update_labels(labels = nil)
     self.label_list = labels
-    self.save!
+    save!
   end
 
   def toggle_status
-    if open?
-      self.status = :resolved
-    else
-      self.status = :open
-    end
-    self.save! ? true : false
+    self.status = if open?
+                    :resolved
+                  else
+                    :open
+                  end
+    save! ? true : false
```

Comment:
save! would either return true or throw an error.  So there isn't much point in having the true or false statement here 

```suggestion
    save! 
```